packages/jsii-calc/lib/compliance.ts

```diff
@@ -2266,3 +2266,18 @@ export class Demonstrate982 {
 
     public constructor() { }
 }
+
+/**
+ * This tries to confuse Jackson by having overloaded property setters.
+ *
+ * @see https://github.com/aws/aws-cdk/issues/4080
+ */
+export class ConfusingToJackson {
```

Comment:
Are we okay with serialization of structs that contain unions, and do we have a test for that?

Reply:
They're _also_ passed by reference; but let me add a test to this effect, still.